app/src/forms/rbac/controller.js

```diff
@@ -85,7 +86,13 @@ module.exports = {
   },
   setSubmissionUserPermissions:  async (req, res, next) => {
     try {
+      const submission = await formService.read(req.query.formSubmissionId, req.currentUser);
       const response = await service.modifySubmissionUser(req.query.formSubmissionId, req.query.userId, req.body, req.currentUser);
+
+      JSON.parse(req.query.isAssigned) ?
+        emailService.submissionAssigned(submission.form.id, response[0], req.body.email, req.headers.referer).catch(() => { })
+        :
+        emailService.submissionUnassigned(submission.form.id, response[0], req.body.email, req.headers.referer).catch(() => { });
```

Comment:
Is the email included in the body?

Reply:
Made changes to target the email properly. Also removed the isAssigned and instead am checking for the permissions array being empty or populated with values.